tests/generator/test_args.py

```diff
@@ -0,0 +1,174 @@
+#!/usr/bin/python3
```

Comment:
There is a case for making this file executable but as it is now it won't do anything. So i'd rather you removed the shebang (running just these tests can be done with the runner)